src/vmspawn/vmspawn.c

```diff
@@ -158,6 +190,27 @@ static int parse_argv(int argc, char *argv[]) {
                             return log_error_errno(r, "Failed to parse --qemu-kvm=%s: %m", optarg);
                         break;
 
+                case ARG_QEMU_VSOCK:
+                        r = parse_tristate(optarg, &arg_qemu_vsock);
+                        if (r < 0)
+                                return log_error_errno(r, "Failed to parse --qemu-vsock=%s: %m", optarg);
+                        break;
+
+                case ARG_VSOCK_CID: {
+                        unsigned cid;
+                        if (isempty(optarg))
+                                cid = VMADDR_CID_ANY;
+                        else {
+                                r = safe_atou_bounded(optarg, 3, UINT_MAX - 1, &cid);
+                                if (r == -ERANGE)
+                                        return log_error_errno(r, "Invalid valid --vsock-cid: %m");
```

Comment:
in log messages and docs please always suffix switches that take parameters with `=`, i.e. here too, please change to: `--vsock-cid=`

Also, "invalid valid"?

Reply:
Fixed in 73418f96ff.